copier/main.py

```diff
@@ -337,16 +363,21 @@ def answers(self) -> AnswersMap:
             )
         for question in questions:
             # Display TUI and ask user interactively only without --defaults
-            new_answer = (
-                question.get_default()
-                if self.defaults
-                else unsafe_prompt(
-                    question.get_questionary_structure(), answers=result.combined
-                )[question.var_name]
-            )
+            try:
+                new_answer = (
+                    question.get_default()
+                    if self.defaults
+                    else unsafe_prompt(
+                        question.get_questionary_structure(), answers=result.combined
+                    )[question.var_name]
+                )
+            except KeyboardInterrupt:
+                raise CopierAnswersInterrupt(result, question, self.template)
```

Comment:
Right now in this case, the last question is nicely grayed out on the TUI. By changing the exception I'm afraid if it would explode on the user hands uglyly. Is that controlled?

Reply:
I'm modifying the CopierAnswersInterrupt to extend KeyboardInterrupt to address this. Will add a test case, but local putzing shows it works as intended. Stacktrace without, No stackstrace with.